lib/surgex/repo_helpers/repo_helpers.ex

```diff
@@ -63,4 +67,22 @@ defmodule Surgex.RepoHelpers do
       opts
     end
   end
+
+  @doc """
+  Sets repo database ssl enable from specified env var.
+  """
+  def set_ssl(opts, env) do
+    if System.get_env(env) do
```

Comment:
Unless env is set, this function will return `nil` (this is due to the fact that it's inside `if` block). This will prevent from chaining other functions after this one is called. I'd go for:
```
with env_value when is_binary(env_value) <- System.get_env(env),
     ssl_value <- String.downcase(env_value) do
  if ssl_value == "true" do
    Keyword.put(opts, :ssl, true)
  else
    Keyword.put(opts, :ssl, false)
  end
else
  _ -> opts
end
```

Reply:
optionally instead of `if` here you could also use
```
case ssl_value do
  "true" -> Keyword.put(opts, :ssl, true)
  "false" -> Keyword.put(opts, :ssl, false)
  _ -> opts
end
```
if you would like not to consider strings other than `true` or `false` valid